### PR TITLE
Separate Ops monitoring files into another tarball. Fixes #5153 in CRABServer

### LINKED ISSUE #5153: complete WMCore cleanup
- [x] 1. is this file still needed ? 
https://github.com/dmwm/CRABClient/blob/master/src/python/CRABClient/WMCoreConfigWrapper.py

- [x] 2. Maybe move the comments at the top to https://github.com/dmwm/CRABClient/blob/master/src/python/CRABClient/WMCoreConfiguration.py

- [x] 3. and make sure to refer to, and explain https://github.com/cms-sw/cmsdist/blob/a3a1ae367973de9b5c57de3b52d8d4df27611b91/crab-build.file#L42-L45

- [x] 4. Must also remove any reference to WMCore still present in https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_12_3_X/master/crab-build.file e.g. lines 507, 16, 23

### PR DESCRIPTION
**Pull request by emaszs**
Issue discussed on crabserver: https://github.com/dmwm/CRABServer/issues/5153

Pull request on crabserver: https://github.com/dmwm/CRABServer/pull/5236

### CODE REVIEW

File: src/python/CRABClient/JobType/UserTarball.py [hunk 1]
```diff
@@ -90,8 +90,20 @@ def addFiles(self, userFiles=None, cfgOutputName=None):
...
+        #configtmp = tempfile.NamedTemporaryFile(delete=True)
+        #configtmp.write(str(self.config))
+        #configtmp.flush()
+        #psetfilename = getattr(self.config.JobType, 'psetName', None)
+        #if not psetfilename == None:
+        #    self.tarfile.add(psetfilename,'/debug/originalPSet.py')
+        #else:
+        #    self.logger.debug('Failed to add pset to tarball')
+        #self.tarfile.add(configtmp.name, '/debug/crabConfig.py')
+        #configtmp.close()
```
Comment (mmascher) [inline src/python/CRABClient/JobType/UserTarball.py]: why did you add these comments?

Comment (emaszs) [inline src/python/CRABClient/JobType/UserTarball.py]: Actually you commented on outdated code, I force-pushed the cleaned up version of this file after the PR was made. Should probably start mentioning that in the comments.
